virttest/utils_test/libvirt.py

```diff
@@ -1541,6 +1541,93 @@ def create_redirdev_xml(redir_type="spicevmc", redir_bus="usb",
     return redir.xml
 
 
+def set_vm_attrs(vmxml, vm_attrs):
```

Comment:
This is not specific libvirt feature related. So I recommend we put it into a new file, like virttest/utils_libvirt/libvirt_vmxml.py.
BTW, libvirt.py is too big and messed up, so we decide not to add new utility functions in it any more.

Reply:
This is for vm xml setting also. Just put it in libvirt_cpu.py also. If it's needed in the future, it can be moved to other place.

Reply:
@jingqi7  I agree with @dzhengfy , how about moving set_vm_attrs to virttest/utils_libvirt/libvirt_vmxml.py now? We already foresee the changes so I think it's the right time

Reply:
Updated

Reply:
UPdated